src/public/app/player/player.component.ts

```diff
@@ -1,38 +1,53 @@
-import { Component, Input, OnChanges } from "angular2/core";
-import {ScPlayer} from "../soundcloud/soundcloud";
-import SoundcloudConnect from "../soundcloud/soundcloud.service";
-import {Song} from "../soundcloud/Song";
-
-const template = require("./player.html");
-import "./player.scss";
+import { Component, Input, OnChanges, SimpleChange } from 'angular2/core';
+import {ScPlayer} from '../soundcloud/soundcloud';
+import SoundcloudConnect from '../soundcloud/soundcloud.service';
+import {Song} from '../soundcloud/Song';
+
+const template: string = `
+<div class="row">
+    <div class="col s6">
+        {{song.title  || "None selected"}}
+    </div>
+    <div class="col s6">
+         <i
+            (click)="stop()"
+            class="small material-icons pointer">stop</i>
+
+        <i
+        (click)="play()"
+        class="small material-icons pointer">play_arrow</i>
+
+    </div>
+</div>
+`;
 
 @Component({
     template,
-    selector: "player"
+    selector: 'player',
 })
-export default class Player implements OnChanges {
+export class Player implements OnChanges {
```

Comment:
(from gchat)
We're gonna do 
imports

class Class {
// definition
}

export Class;
export default OtherClass;